docs/source/_comparison_generator.py

```diff
@@ -107,6 +107,9 @@ def generate():
     buf += _section(
         'Sparse Matrices',
         'scipy.sparse', 'cupyx.scipy.sparse', 'SciPy')
+    buf += _section(
```

Comment:
Would you move this section before "Sparse Matrices" to follow SciPy's API reference list if you don't have a special reason?

https://docs.scipy.org/doc/scipy/reference/index.html#api-reference